p4rmyknife/modules/config.py

```diff
@@ -16,3 +16,13 @@ def install_client(args):
        p4_url = 'https://louis-test-p4k.s3.us-east-2.amazonaws.com/TestFile.sh'
       #urllib.request.urlretrieve(p4_url, 'p4v.tgz')
        urllib.request.urlretrieve(p4_url, 'TestFile.sh')
+
+    #Need to write script that will check for the downloaded file- 
+    #Testfile.sh, then run it, if present, or produce error if not present.
+    
+def check_file(args):
```

Comment:
While you _can_ move this to it's own function, it's so simple that I don't think it gains you much... At least not until you see yourself having to do this multiple times.

Reply:
Ok. Since my goal for this piece of code is probably only going to be run once, I see what you mean that there's no need to commit it to a function.